src/Elmish.WPF/Binding.fs

```diff
@@ -143,20 +143,20 @@ module internal BindingData =
       }
     | TwoWayData d -> TwoWayData {
         Get = unbox >> d.Get
-        Set = fun v m -> d.Set v (unbox m) |> box
```

Comment:
To be honest I like the existing forms better for all changes in this file. All the boxing and unboxing is difficult enough to grasp without also using point-free style to remove some parameters of multi-parameter functions. I'd like to keep these multi-parameter functions explicitly as multi-parameter functions. Also see later comment on multi-parameter functions where the parameters are of "equal importance", as it were.

Reply:
Sure.  Sometimes point-free goes too far.  This is probably one of those times.

At the same time, I think putting `unbox` on the left helps to verify correctness.  That is the first hint I had that #116 might be a bug.  Getters should look like `unbox >> get` and setters should look like `unbox >> set >> box`.

In any case, this seems complicated enough that I have reverted all the changes in this file for this PR.  If I refactor this code to put it in a standard form (like above), then it will be in a separate PR related to #116.

Reply:
Thanks! I get your point regarding `unbox >> set >> box`, though I think I still find that the current form is more easily understood.